src/fileAction.js

```diff
@@ -0,0 +1,14 @@
+const alfy = require("alfy");
+
+// check node version
+const majorVersion = Number(process.version.replace("v", "").split(".")[0]);
+if (majorVersion < 8) {
+  alfy.error(`Wrong Node version. We need v8+, you have ${process.version}`);
+  return;
+}
+
+const product = require("./product").get();
+const query = alfy.input;
+
+// output
+console.log(product.binPath + ' "' + query + '"');
```

Comment:
Maybe it's more clean if we use `alfy.output` and create an "item". What do you think?

Reply:
but utilising `alfy.output` actually complicate things, 

[link](https://github.com/sindresorhus/alfy/blob/10a1a7357332b89c3866b36e6a247efad39fcc6c/index.js#L40) output function will nest my only `item` in an array, which I need to access it through `zsh`.

Reply:
@bchatard can you show me the better example?

Reply:
if you output the same format as "standard" workflow, you can reuse the existing *run script*, no?
If yes, your previous commit with the export of `buildItem` is a good way to achieve this. And also fix the missing notification.

(a lot of guessing on this, I'm not really available for more debugging on this actually)

Reply:
![image](https://user-images.githubusercontent.com/829364/61775373-232b1a00-adf9-11e9-90a0-f49b450a7890.png)

Reply:
Yup, agree, currently the Notification does not show proper message. Will get back to this next weekend. Not available at the moment.